open-craft/opencraft
# Don't set COMMON_HOSTNAME to instance domain.

`COMMON_HOSTNAME` is mapped to `127.0.0.1` on edxapp app servers, making the instance inaccessible from the app server itself.

This patch uses a different value for `COMMON_HOSTNAME`, derived from the domain and the appserver name. The value itself probably  doesn't matter much.

**Testing instructions**:

Spawn a new appserver, SSH into it, and verify that `/etc/hostname` is set to a value derived from the instance domain and the app server name, instead of the instance domain.
